packages/firmware_notifier/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## 0.0.1
+
+- Initial version.
```

Comment:
we can leave this stuff out because it's an internal tool bundled with the firmware-updater :)

Reply:
:+1: